<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BranchesManage</class>
 <widget class="QWidget" name="BranchesManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Branches Manage</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>301</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Address</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Branch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>94</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Branch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>94</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>BranchesManage</receiver>
   <slot>onAddClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>BranchesManage</receiver>
   <slot>onDeleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>BranchesManage</receiver>
   <slot>onApplyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>BranchesManage</receiver>
   <slot>onResetClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddClicked()</slot>
  <slot>onDeleteClicked()</slot>
  <slot>onApplyClicked()</slot>
  <slot>onResetClicked()</slot>
 </slots>
</ui>
